_posts/2016-06-20-alpha-release-notes.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "argparse2cwl alpha release notes"
+date:   2016-06-20 02:27:27 +0300
+comments: true
+---
+
+`argparse2cwl` forms CWL tool descriptions from programs which use `argparse` as their argument parser. 
+
+## Prerequisites ##
+`argparse2cwl` is compatible with both Python 2 & 3.
+Dependencies:
+* Jinja2 >= 2.8
+
+## Installing ##
+
+Installation process:
+
+1. Clone the source repository:
```

Comment:
@anton-khodak, if that `gxargparse_check_path` is really needed at the end, would it be possible to run it on install time instead of having the user to run it manually?